.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I'm not sure this should be committed to source since it's IDE specific?

Reply:
No, this should not have been commited, good catch.